Networking/Networking/Model/Refund/Refund.swift

```diff
@@ -0,0 +1,59 @@
+import Foundation
+
+/// Represents a Refund request Entity
+///
+public struct Refund {
+
+    /// Total amount to be refunded
+    ///
+    public let amount: String
+
+    /// Optional refund reason
+    ///
+    public let reason: String?
+
+    /// If true, the automatic refund is used. When false, manual refund process is used.
+    ///
+    public let apiRefund: Bool
+
+    /// Optional Line items to be refunded
+    ///
+    public let items: [LineItemRefund]?
+
+    public init(amount: String, reason: String?, apiRefund: Bool = true, items: [LineItemRefund]?) {
+        self.amount = amount
+        self.reason = reason
+        self.items = items
+        self.apiRefund = apiRefund
+    }
+
+    public func toDictionary() -> [String: Any] {
+        var dict: [String: Any] = ["amount": amount, "api_refund": apiRefund]
```

Comment:
Same comment about the keys. Would it make sense to declare them in a private enum / struct?